LandSandBoat/server
# [sql/lua] Nommo NM

**_I affirm:_**

## What does this pull request do?

Implements [Nommo](https://www.bg-wiki.com/ffxi/Nommo)

- Updates the behavior for the NM Nommo. Verified in retail.
- Updates the HP of Nommo. Verified in retail.
- Updates the drop rate of Nommo, Lyft Lance to something more realistic and standard.
- Fixes the drop from Tikbalang. There was a typo where the Lyft Lance was duplicated by accident. It should be Lyft Pole.
- Adds another mob AE for enamnesia

## Steps to test these changes

set PH to spawn him 100% of the time: `xi.mob.phOnDespawn(mob, ID.mob.NOMMO_PH, 100, 3600, { immediate = true }) -- 1 hour`

kill PH

fight Nommo, see it does stuff
![image](https://github.com/LandSandBoat/server/assets/131182600/d5932705-0f31-4f80-a151-0ccba3931e85)
